code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -90,6 +91,7 @@ This file's folder contains:
 	ticker.mode.servants_of_ratvar -= M.mind
 	ticker.mode.update_servant_icons_removed(M.mind)
 	all_clockwork_mobs -= M
+	M.faction -= "ratvar"
```

Comment:
Fun(?) fact; this won't work, because for some reason factions are an assoc list(that never use anything but the keys)